polygeist/lib/Dialect/Polygeist/Transforms/LICM.cpp

```diff
@@ -96,24 +96,6 @@ class OperationSideEffects {
   /// otherwise.
   bool conflictsWith(const Operation &other) const;
 
-  /// Returns the first operation in the given block \p block with side effects
-  /// that conflict with the side effects summarized in this class, and a
-  /// std::nullopt if none is found. If \p point is given, only operations that
-  /// appear before it are considered.
-  Optional<Operation *>
-  conflictsWithOperationInBlock(Block &block, Operation *point = nullptr) const;
-
-  /// Returns the first operation in the given region \p rgn with side effects
-  /// that conflict with the side effects summarized in this class, and a
-  /// std::nullopt if none is found.
-  Optional<Operation *> conflictsWithOperationInRegion(Region &rgn) const;
-
-  /// Returns the first operation in the given loop \p loop with side effects
-  /// that conflict with the side effects summarized in this class, and a
-  /// std::nullopt if none is found.
-  Optional<Operation *>
-  conflictsWithOperationInLoop(LoopLikeOpInterface loop) const;
```

Comment:
`conflictsWithOperationInRegion` and `conflictsWithOperationInLoop` are not used.